libcontainerd/remote_linux.go

```diff
@@ -352,7 +352,9 @@ func (r *remote) runContainerdDaemon() error {
 		logrus.Debugf("runContainerdDaemon: runtimeArgs: %s", args)
 	}
 	cmd := exec.Command(containerdBinary, args...)
-	// TODO: store logs?
+	// redirect containerd logs to docker logs
+	cmd.Stdout = os.Stdout
```

Comment:
is this output always true? I'm not sure right now if we allow people to change this output stream.